SECURITY-RELEASE-PROCESS.md

```diff
@@ -0,0 +1,188 @@
+# Security Release Process
+
+CoreDNS project has adopted this security disclosures and response policy
+to ensure a responsible handle of critical issues.
+
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. 
+The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal communication and external disclosure.
+
+The initial Product Security Team will consist of set of maintainers that volunteered.
+The list is available in this [document](/OWNERS) and is synchronized with the receivers of product-security-team@coredns.io list.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+If you find a security vulnerability or any security related issues, 
+please DO NOT file a public issue - that means do not create a Github issue, 
```

Comment:
Rephrase this to:

If you find a security vulnerability or any security related issues,
please DO NOT file a public issue. Do not create a GitHub issue.
Instead, send your report privately to ...

Also, is that email address supposed to be the same as "product-security-team" or is it really a different address?